presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -232,6 +233,8 @@
     public static final String SEGMENTED_AGGREGATION_ENABLED = "segmented_aggregation_enabled";
     public static final String USE_HISTORY_BASED_PLAN_STATISTICS = "use_history_based_plan_statistics";
     public static final String TRACK_HISTORY_BASED_PLAN_STATISTICS = "track_history_based_plan_statistics";
+    public static final String MAX_SCAN_NODE_IN_PLAN = "max_scan_node_in_plan";
+    public static final String PLAN_NODE_LIMIT_ENABLED = "plan_node_limit_enabled";
```

Comment:
call this SCAN_NODE_LIMIT_ENABLED

Reply:
Changed to `LEAF_NODE_LIMIT_ENABLED` as we include both Value and Scan nodes.